modules/ROOT/partials/glossary.adoc

```diff
@@ -1,117 +1,132 @@
 [glossary]
 
 [[action]]$action:: Freeform description of an action.
-[[alias]]$alias:: An alias name.
-[[alloc]]$alloc:: Allocator name.
-[[allocType]]$allocType:: Allocation type.
-[[auth]]$auth:: Auth provider name.
-[[boltMsgLenLimit]]$boltMsgLenLimit:: Bolt message length limit.
-[[boltServerState]]$boltServerState:: Bolt server state.
+//$action will be replaced by cmd and msg depending on the usage.
+[[alias]]$alias:: An alias name, `mydb-alias`.
+[[allocType]]$allocType:: Allocation type (`primary` or `secondary`).
+[[arg]]$arg:: Procedure or function argument, for example, `database`, `pause`, `dryrun`, etc.
+[[argType]]$argType:: Argument type, either input or output.
+[[auth]]$auth:: Auth provider name, for example, `ldap`, `native`.
+[[boltServerState]]$boltServerState:: Bolt server state, for example, `READY`.
+[[boltMsgLenLimit]]$boltMsgLenLimit:: Bolt message length limit, for example, `1000`.
 [[cause]]$cause:: Freeform description of what caused the error.
 [[cfgSetting]]$cfgSetting:: Configuration setting key, for example, `server.https.enabled`.
-[[clause]]$clause:: A clause, for example, `USE`.
-[[cmd]]$cmd:: Cypher administrative command, for example, `DROP DATABASE`.
-[[component]]$component:: Component name.
-[[constr]]$constr:: Constraint name.
-[[constrDescrOrName]]$constrDescrOrName:: Constraint descriptor or name.
-[[context]]$context:: Freeform description of a context.
-[[coordinates]]$coordinates:: Coordinate value.
-[[count]]$count:: Amount
+[[clause]]$clause:: A Cypher clause, for example, `USE`, `MATCH`.
+[[cmd]]$cmd:: A Cypher command, for example, `DROP DATABASE`, `PROFILE`, `SHOW PROCEDURES`.
+[[component]]$component:: Data type component, for example, `year`, `month`, `hour`, etc.
```

Comment:
I am not sure if this was intentional or if it is a typo and should be 'date type component'. At least all our examples are related to temporals, but we maybe wanted to be more general. What do you think?

Reply:
In the spreadsheet, it's a component name. But I think it makes more sense to use a date type component, as the data type component might be misinterpreted by a component related to data types.